modules/ROOT/pages/manage-proxy.adoc

```diff
@@ -31,29 +31,18 @@ The output should match the value expected.
 
 == Update the HTTP Proxy
 
-This operation requires two procedures: 
+Updating the HTTP proxy causes each machine in the cluster to restart all pods running on it. Although no downtime occurs if you are running multiple replicas, you should perform HTTP proxy updates during a maintenance window.
 
-* Run a script on each node of your Runtime Fabric to update the cluster.
 * Apply the proxy values to the services running within Runtime Fabric.
-
-=== Run the Script on Each Node
-
-. Copy the `set_dockerd_proxy.sh` from the controller node used to install Runtime Fabric, which is located under `/opt/anypoint/runtimefabric/scripts/`, to each controller and worker node under the same location (`/opt/anypoint/runtimefabric/scripts/`). +
-[NOTE]
-If you are not able to locate this script, please open a support ticket for assistance.
-. Update the `/opt/anypoint/runtimefabric/env` file to set the values for `RTF_HTTP_PROXY` and `RTF_NO_PROXY` variables. + 
-The `RTF_HTTP_PROXY` value accepts authentication values in-line. For example: `http://user:Password@1.1.1.1:80` +
-The `RTF_NO_PROXY` value accepts a comma-separated list of hosts that should not go through the proxy. For example: `1.1.1.1,no-proxy.com`
-. Run the script `/opt/anypoint/runtimefabric/scripts/set_dockerd_proxy.sh`.
-. After repeating these steps on each controller and worker node, apply these proxy values to the services running within Runtime Fabric.
+* If you are using Runtime Fabric installer 1.0.x, run the script on each node of your Runtime Fabric to update the cluster.
```

Comment:
```suggestion
* If you are using Runtime Fabric installer 1.0.x, run the script on each node of your fabric to update the cluster.
```
Names like "Runtime Fabric" require a noun object; they are not themselves nouns.